packages/commonwealth/client/scripts/views/pages/user_dashboard/user_dashboard_chain_event_row.tsx

```diff
@@ -48,7 +48,7 @@ export const UserDashboardChainEventRow = (
             onClick={(e) => {
               e.preventDefault();
               e.stopPropagation();
-              navigate(`/${chain}`);
+              navigate(`/${chain?.id}`);
```

Comment:
when I click avve, quickly go back and click avve again, I am gettin `/avve/avve`. 

https://user-images.githubusercontent.com/14819225/225886434-37cbc6bd-ff34-4b1a-a436-1fcbb7eda838.mov


So I think it should be `navigate('/${chain?.id}, {}, null');`

![image](https://user-images.githubusercontent.com/14819225/225885858-2b432115-b53c-4ff7-827e-b9ffb51a8864.png)